src/python/pants/backend/go/goals/tailor.py

```diff
@@ -79,7 +79,7 @@ async def find_putative_go_targets(
                 PutativeTarget.for_target_type(
                     GoPackageTarget,
                     path=dirname,
-                    name=None,
+                    name="pkg",
```

Comment:
This name is not temporary, per the proposal at https://docs.google.com/document/d/1N1bX2PsEHL-Ue9_Spsl3NlyHxmgKtvpB5y4RAD-cOZQ/edit#. A `helm_chart` is an "atom target" so the new shortcut for generated targets won't apply.

The address will look like `path/to:chart`.

Reply:
Should this be `go_pkg`?